pkg/assembler/backends/ent/backend/vulnerability.go

```diff
@@ -42,23 +45,34 @@ func (b *EntBackend) IngestVulnerabilities(ctx context.Context, vulns []*model.V
 	return modelVulnerabilities, nil
 }
 
-func (b *EntBackend) IngestVulnerability(ctx context.Context, vuln model.VulnerabilityInputSpec) (*model.Vulnerability, error) {
-	record, err := WithinTX(ctx, b.client, func(ctx context.Context) (*int, error) {
-		client := ent.TxFromContext(ctx)
-		vulnerabilityID, err := upsertVulnerability(ctx, client, vuln)
-		if err != nil {
-			return nil, err
-		}
+func (b *EntBackend) IngestVulnerabilityID(ctx context.Context, vuln model.VulnerabilityInputSpec) (string, error) {
+	id, err := WithinTX(ctx, b.client, func(ctx context.Context) (*int, error) {
+		return upsertVulnerability(ctx, ent.TxFromContext(ctx), vuln)
+	})
+	if err != nil {
+		return "", err
+	}
+
+	return strconv.Itoa(*id), nil
+}
 
-		return vulnerabilityID, nil
+func (b *EntBackend) IngestVulnerabilityIDs(ctx context.Context, vulns []*model.VulnerabilityInputSpec) ([]string, error) {
+	ids, err := WithinTX(ctx, b.client, func(ctx context.Context) (*[]string, error) {
```

Comment:
I would consider having a kind of consistency with previous implementation and the bulk operation implementations for other resources based on the `for _, vuln := range vulns` loop.
Just to have things done  (almost) in the same way across endpoints.

Reply:
ok